File: ABC/062/a_ans.py
def main():
    # input
    x, y = map(int, input().split())

    # compute
    lss = [0, 1, 3, 1, 2, 1, 2, 1, 1, 2, 1, 2, 1]

    # output
    print('Yes' if lss[x]==lss[y] else 'No')


if __name__ == '__main__':
    main()

File: abc/162/a_ans.c
#include <stdio.h>
#include <string.h>

int main(void) {
  // input
  int N; scanf("%d", &N);

  // compute
  int a = N / 100;
  int b = N % 100 / 10;
  int c = N % 10;

  // output
  if (a==7 || b==7 || c==7) {
    printf("Yes\n");
  }
  else {
    printf("No\n");
  }
  return 0;
}
